packages/flutter_tools/lib/src/test/flutter_web_goldens.dart

```diff
@@ -157,7 +158,7 @@ import 'package:flutter_test/flutter_test.dart';
 ${testConfigFile != null ? "import '${Uri.file(testConfigFile.path)}' as test_config;" : ""}
 
 void main() async {
-  LocalFileComparator comparator = LocalFileComparator(Uri.parse('$testUri'));
+  LocalFileComparator comparator = LocalFileComparator(Uri.parse('$embeddedUrl'));
```

Comment:
We don't run the web tests on windows, but this URI needs to be encoded due to `\` in windows file paths.